rental.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'book'
+require_relative 'person'
+class Rental
+  def initialize(date)
+    @date = date
+    @book = nil
+    @person = nil
+  end
```

Comment:
- [ ] Please add the `book` and `person` to the parameters of the constructor of the `Rental` class. The `Rental` class should look something like the following:
```ruby
class Rental

   ...

   def initialize(date, book, person)
   
   ...
   
   @book = book
   book.rentals << self

   @person = person
   person.rentals << self
  end
  
  ...
  
end

```